### PR TITLE
DOCS Add repo dependency for CKAN registry

### PR DESCRIPTION
**Pull request by sachajudd**
To be merged once CKAN registry has been tagged. cc @ScopeyNZ 

Relies on: https://github.com/silverstripe/silverstripe-userhelp-content/pull/133

### CODE REVIEW

File: app/_config/docs-repositories.yml [hunk 1]
```diff
@@ -162,3 +162,7 @@ RefreshMarkdownTask:
       - silverstripe/silverstripe-documentconverter
       - silverstripe-documentconverter
       - master
+    -
+      - silverstripe/ckan-registry
+      - silverstripe/ckan-registry
```
Comment (ScopeyNZ) [inline app/_config/docs-repositories.yml]: I know nothing about this config but is this line meant to be duplicated?

File: app/_config/docs-repositories.yml [hunk 2]
```diff
@@ -162,3 +162,7 @@ RefreshMarkdownTask:
       - silverstripe/silverstripe-documentconverter
       - silverstripe-documentconverter
       - master
+    -
+      - silverstripe/ckan-registry
```
Comment (robbieaverill) [inline app/_config/docs-repositories.yml]: This looks like a GitHub slug to me - maybe should be `silverstripe/silverstripe-ckan-registry`?

File: app/_config/docs-repositories.yml [hunk 3]
```diff
@@ -162,3 +162,7 @@ RefreshMarkdownTask:
...
+      - ckan-registry
```
Comment (robbieaverill) [inline app/_config/docs-repositories.yml]:
```suggestion
      - silverstripe-ckan-registry
```

For consistency with the other modules

Comment (sachajudd) [reply app/_config/docs-repositories.yml]: A lot of the modules added here are named like this e.g 'elemental', 'taxonomy' and 'dsm'. This is just the folder name that the symlink pulls in, see https://github.com/silverstripe/silverstripe-userhelp-content/pull/133/files. I'll make a PR to change the inconsistent reqs (maintenance, sharedraftcontent, documentconverter).

File: app/_config/docs-repositories.yml (hunk 2 above)
Comment (sachajudd) [reply app/_config/docs-repositories.yml]: Will update.

**Pull request merged**